src/Sulu/Bundle/MediaBundle/Media/FormatManager/FormatManager.php

```diff
@@ -461,6 +511,7 @@ protected function getImageExtension($fileName)
         switch ($extension) {
             case 'png':
             case 'gif':
+            case 'jpeg':
```

Comment:
To be honest I don't really know why this is need. I touched the code because it was an issue that `jpeg` was converted to `jpg`.